tools-local/prebuilt-baseline-offline.xml

```diff
@@ -208,6 +196,7 @@
     <Usage Id="Microsoft.NETCore.Platforms" Version="2.1.2" />
     <Usage Id="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0-alpha-5" IsDirectDependency="true" />
     <Usage Id="Microsoft.NETFramework.ReferenceAssemblies.net40" Version="1.0.0-alpha-5" />
+    <Usage Id="Microsoft.NETFramework.ReferenceAssemblies.net40" Version="1.0.0-preview.2" />
```

Comment:
I was expecting Microsoft.Web.Xdt.3.0.0 to go away as a prebuilt.

Reply:
Xdt can be built from source. What needs to happen for this?